core/src/main/java/org/eclipse/hono/config/ServiceConfigProperties.java

```diff
@@ -205,4 +212,32 @@ public final String getCorsAllowedOrigin() {
     public final void setCorsAllowedOrigin(final String corsAllowedOrigin) {
         this.corsAllowedOrigin = Objects.requireNonNull(corsAllowedOrigin);
     }
+
+    /**
+     * Gets the send timeout value in milliseconds, which is to be used when sending a 
+     * message on the vert.x event bus.
+     * <p>
+     * The default value of this property is {@link #DEFAULT_SEND_TIMEOUT_IN_MS}.
+     * 
+     * @return The send timeout value in milliseconds.
+     */
+    public final long getSendTimeOut() {
+        return sendTimeOutInMs;
+    }
+
+    /**
+     * Sets the send timeout value in milliseconds, which is to be used when sending a 
+     * message on the vert.x event bus.
+     * <p>
+     * The default value of this property is {@link #DEFAULT_SEND_TIMEOUT_IN_MS}.
+     * 
+     * @param sendTimeOutInMs The send timeout value in milliseconds.
+     * @throws IllegalArgumentException if the timeout is negative.
+     */
+    public final void setSendTimeOut(final long sendTimeOutInMs) {
+        if (sendTimeOutInMs < 0) {
+            throw new IllegalArgumentException("send time out value must not be negative");
```

Comment:
does it make sense to set this to 0?

Reply:
Just I checked the implementation of `DeliveryOptions.setSendTimeout(long timeout)`. They throw `IllegalArgumentException`, if the value is _< 1_ but its not documented in that method's _javadoc_. Anyway it cannot be set to 0. I will update it.

Reply:
My point is actually that IMHO it doesn't make sense to accept a very small value here, i.e. we should enforce a minimum value that makes some sense, e.g. 500ms?

Reply:
I see now what do you mean. But how do we conclude for such a minimum value?  Anyway we set a default value of 3000ms. `DeliveryOptions.setSendTimeout(...)` allows to set any timeout value greater than 0ms. IMHO why should we forbid this flexibility by setting a minimum value?

Reply:
Well, as I pointed out: because the system will not work if you set it to 1ms, will it?

Reply:
> Well, as I pointed out: because the system will not work if you set it to 1ms, will it?

Ok I see the point. The systems commonly we use are not yet so fast enough to set the value to 1 ms ;-). I will update it to a minimum value of 500ms.

Reply:
I have updated it with a minimum value of _500ms_ and pushed the changes. Would you mind taking a look?